docs/source/installation.rst

```diff
@@ -126,68 +146,37 @@ the command below would install GPU packages required for ASPIRE.
 
 ::
 
-    # From PyPI
-    pip install -e "aspire[gpu_11x]"
-
     # From a local git repo
     pip install -e ".[gpu_11x]"
+
+    # From PyPI
+    pip install "aspire[gpu_11x]"
+
     
-By default if GPU extensions are correctly installed,
+By default if the required GPU extensions are correctly installed,
 ASPIRE should automatically begin using the GPU for select components
 (such as those using ``nufft``).
 
+Because GPU extensions depend on several third party packages and
+libraries, we can only offer limited support if one of the packages
+has a problem on your system.
+
 Generating Documentation
 ************************
 
-Sphinx Documentation of the source (a local copy of what you're looking at right now) can be generated using:
+Sphinx Documentation of the source (a local copy of what you're
+looking at right now) can be generated by using the following commands
+from the root of the code repository.
+
+The ``make html`` command runs and renders the ``gallery/tutorials``
+examples, which takes several minutes.
 
 ::
 
-    cd /path/to/git/clone/folder/docs
+    cd docs
     sphinx-apidoc -f -o ./source ../src -H Modules
     make clean
-    make html
-
-The built html files can be found at ``/path/to/git/clone/folder/docs/build/html``
-
-
-Optimized Numerical Backends
-****************************
-
-Conda provides optimized numerical backends that can provide significant
-performance improvements on appropriate machines.  The backends accelerate
-the performance of ``numpy``, ``scipy``, and ``scikit`` packages.
-ASPIRE ships several ``environment*.yml`` files which define tested package
-versions along with these optimized numerical installations.
-
-The default ``environment-default.yml`` does not force a specific backend,
-instead relying on ``conda`` to select something reasonable.
-In the case of an Intel machine, the default ``conda`` install
-will automatically install some optimizations for you.
-However, these files can be used to specify a specific setup
-or as the basis for your own customized ``conda`` environment.
-
-.. list-table:: Suggested Conda Environments
-   :widths: 25 25
-   :header-rows: 1
+    make html-noplot  # Generate only documentation
```

Comment:
The way this is presented, users may think that you have to run both `make html-noplot` followed by `make html`. Should make clear that these are just two alternatives.

Reply:
Sure! Added some more detail.